app/views/mentors/index.html.erb

```diff
@@ -1,27 +1,28 @@
-<h1>Mentors</h1>
-
 <!-- Mentor Table -->
-<table class='highlight responsive-table'>
-  <thead>
-    <tr>
-      <th>Image</th>
-      <th>Name</th>
-      <th>Company</th>
-      <th>Specialty</th>
-      <th>Email</th>
-      <th>Phone</th>
-    </tr>
-  </thead>
-  <tbody>
-    <% 15.times do %>   <!-- this will be changed once database created -->
+<container>
+  <table class='highlight responsive-table'>
```

Comment:
yes and yes -- good idea with wrapping the yield in a container and resizing it. I think I must have erased my container or something before I pushed because what was there made no sense.